QuizApp/QuizSession/AnswersView.swift

```diff
@@ -0,0 +1,38 @@
+import SwiftUI
+
+struct AnswersView: View {
+
+    let correctAnswerIndex: Int
+    let nextQuestion: (Int, Color) -> Void
+
+    @Binding var numberOfCorrectQuestions: Int
+    @Binding var answers: [AnswerModel]
+
+    var body: some View {
+        VStack(spacing: 10) {
+            ForEach(answers) { answer in
+                Button {
+                    if answer.index == correctAnswerIndex {
+                        numberOfCorrectQuestions += 1
+                        answers[answer.index].color = .green
+                        nextQuestion(numberOfCorrectQuestions, .green)
+                    } else {
+                        answers[answer.index].color = .red
+                        answers[correctAnswerIndex].color = .green
+                        nextQuestion(numberOfCorrectQuestions, .red)
+                    }
```

Comment:
I would move the body of `if` and body of `else` to a new function. SwiftUI is kind of hard to read when colsures are used so having sth like:
```swift
Button {
    if answer.index == correctAnswerIndex {
        correctAnswerSelected()
    } else {
       incorrectAnswerSelected()
    }
}
```